synapse/lib/cell.py

```diff
@@ -496,6 +534,10 @@ class Cell(s_nexus.Pusher, s_telepath.Aware):
             'description': 'A config-driven way to specify the HTTPS port.',
             'type': ['integer', 'null'],
         },
+        'backup:dir': {
+            'description': 'A directory outside the service directory where backups will be saved.',
+            'type': ['string', 'null'],
```

Comment:
Not sure if this was intended, but multiple types prevent this from being a command line arg

Reply:
it needs to be ok being null, so i guess i'd call it intentional.  out of curiosity, what do we do in the case where null needs to be ok but we do want a command line arg?

Reply:
I believe you can do this:
`        'backup:dir': {
            'description': 'A directory outside the service directory where backups will be saved.',
            'type': 'string',
            'default': None,
        },`